users/autostart.rst

```diff
@@ -198,16 +198,27 @@ On any distribution (Arch, Debian, Linux Mint, Ubuntu, openSUSE)
 Using Supervisord
 ~~~~~~~~~~~~~~~~~
 
-Add the following to your supervisor config file::
+Create the ``syncthing.conf`` file in supervisor's ``conf.d`` folder (the folder is located
+at ``/etc/supervisor/conf.d``)::
```

Comment:
In supervisors x folder which is actually /etc/foo/x, why not just /etc/foo/x straight away?

Reply:
Is it different between ubuntu/debian?

Reply:
Not sure I understand your question. Debian/Ubuntu installation uses `conf.d` subfolder as default location for custom configs which helps keeping all stuff organized.

Reply:
Sure, as long as they are in the same place no point talking about conf.d and then giving the full path, give the full path straight away.

Reply:
Done